source/class.cpp

```diff
@@ -408,6 +408,16 @@ void ClassBinder::request_bindings_and_skipping(Config const &config)
 /// extract include needed for this generator and add it to includes vector
 void ClassBinder::add_relevant_includes(IncludeSet &includes) const
 {
+	string const qualified_name_without_template = standard_name( C->getQualifiedNameAsString() );
+	std::map<string, std::vector<string>> const &per_class_includes= Config::get().per_class_includes();
+	
+	auto pci = per_class_includes.find(qualified_name_without_template);
+	if(pci != per_class_includes.end()) {
+		for(auto const & i : pci->second) {
+			includes.add_include(i);
```

Comment:
Looks good Alex, thank you for refactoring this!

Small note: could you please add `if(O_annotate_includes)` and when annotation is requested add comment `// +include_for_class` for these includes so it is obvious when they come from? (this will be really helpful for any include-debugging later)

https://github.com/RosettaCommons/binder/blob/master/source/type.cpp#L188

Reply:
Good idea, updated along with expected test results.